test/test.js

```diff
@@ -1,8 +1,164 @@
-import assert from 'assert';
-import { describe, it } from 'mocha';
+import { expect } from 'chai';
+import { describe, it, beforeEach, afterEach } from 'mocha';
+import {
+  InteractionResponseType,
+  InteractionType,
+  InteractionResponseFlags,
+} from 'discord-interactions';
+import { AWW_COMMAND, INVITE_COMMAND } from '../src/commands.js';
+import sinon from 'sinon';
+import server from '../src/server.js';
+import reddit from '../src/reddit.js';
 
-describe('awwbot', () => {
-  it('should have tests', () => {
-    assert.ok(true);
+describe('Server', () => {
+  describe('GET /', () => {
+    it('should return a greeting message with the Discord application ID', async () => {
+      const request = {
+        method: 'GET',
+        url: new URL('/', 'http://discordo.example'),
+      };
+      const env = { DISCORD_APPLICATION_ID: '123456789' };
+
+      const response = await server.fetch(request, env);
+      const body = await response.text();
+
+      expect(body).to.equal('👋 123456789');
```

Comment:
This is a total preference thing.... @shaydewael do we have an opinion on `assert.strictEqual` style vs `expect` style throughout our bots?  I typically roll with assert since it avoids the additional module 🤷

Reply:
Love some consistency! @shaydewael any preference here?

Reply:
I admittedly don't have a strong opinion here, especially since it's within tests and not the core app. I'm good with `assert`, but also think having `expect`/an extra package is a hugeeeee deal if it makes it easier or clearer.